packages/amplication-server/src/models/Resource.ts

```diff
@@ -29,9 +29,9 @@ export class Resource {
   })
   updatedAt!: Date;
 
-  workspace?: Workspace;
+  project?: Project;
 
-  workspaceId?: string;
+  projectId?: string;
```

Comment:
why did you create these properties without @Field decorator?

Reply:
I forgot to add. Previously it didn't use fields because it took the values from the context, now the user is providing it. Right?